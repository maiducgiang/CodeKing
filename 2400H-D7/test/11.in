10
33724 5000
96903 5000
76788 5000
61291 5000
77402 5000
12720 5000
33759 5000
67434 5000
41951 5000
74984 5000
