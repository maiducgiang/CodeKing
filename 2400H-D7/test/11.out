749536757
481070571
57512075
303220491
464107654
452825178
795644646
486523621
658066861
98424108
